### DISCUSSION TITLE
Import Accounts from CSV

### DISCUSSION THREAD

**Discussion by jawwadnaeem** [modoboa/modoboa#3176] [Help] [answered]
Hello all,

I need help in importing 15 accounts from CSV file using modoboa web admin panel. Can someone send me the exact format i need to use in CSV file also what needs to be put in separator field

I am using below format in CSV file 

account;faaraea@example.com;{SHA512-CRYPT}$6$rounds=70000$jY44UYsdCtqWPnPsdQNa$pdBYkhs.dwH5X31casdasdrH5EnF1Z4gTbaVCxcMhhgL1U9EH/BL7N2iXK57MJhcNFN5senjpMy8AxKIvS37eMezK/qV1;Dr Faaraea;;TRUE;SimpleUsers;faaraea@example.com

**Comment by ghost** [comment, marked as answer]
[Here](https://modoboa.readthedocs.io/en/latest/moving.html#using-csv-files)

**Reply by linuxmaster** [reply]
Hello, everyone!
is it possible to extract and import csv-data without web-interface? My 2.3.3 installation is broken... My bad.

**Reply by tonioo** [reply]
Yes, there is a command line tool to import data:
```python manage.py modo import ...```
